### ISSUE TITLE
Need to do better versioning for runtime and sdk packages for Linux

### ISSUE THREAD

**Issue by Petermarcu** [dotnet/core-setup#3556]
People take dependencies on our package names. When we update, they end up stuck on the old one and force installation of older packages all the time.

**Comment by Petermarcu** [comment]
@leecow I think you said you have an idea on what this should look like.

**Comment by Petermarcu** [comment]
https://github.com/dotnet/cli/issues/8209 This is the one tracking the same changes needed in the runtime.

**Comment by leecow** [comment]
Yep, meeting with folks this week to begin the planning and path forward.

**Comment by steveharter** [comment]
Removing the area-SDK since we shouldn't have two

**Comment by omajid** [comment]
https://github.com/dotnet/docs/pull/4538 has an updated proposal that should address this
